chocolatey/plugins/modules/win_chocolatey_facts.py

```diff
@@ -18,6 +18,22 @@
 short_description: Create a facts collection for Chocolatey
 description:
 - This module shows information from Chocolatey, such as installed packages, outdated packages, configuration, feature and sources.
+options:
+  gather_subset:
+      version_added: "1.5.0"
+      description:
+          - "If supplied, restrict the facts collected to the given subset.
+            Possible values: C(all), C(config), C(feature), C(outdated),
+            C(packages), C(sources).
+            You can provide a list of values to specify a larger subset.
+            Values can also be used with an initial C(!) to specify that that
+            specific subset should be excluded from collection.  For instance:
+            C(!outdated,!packages).
+            To collect only specific facts, use C(!all), and specify the desired
+            subsets."
+      type: list
```

Comment:
Other modules have the `description` content not indented, so we should probably be consistent... and this can be broken into multiple points:
```suggestion
      description:
      - If supplied, restrict the facts collected to the given subset.
        Possible values: C(all), C(config), C(feature), C(outdated),
        C(packages), C(sources).
      - You can provide a list of values to specify a larger subset.
        Values other than C(all) can also be used with an initial C(!)
        to specify that that specific subset should be excluded from
        collection. For instance: C(!outdated,!packages).
      type: list
```

Note that part of this also ties into what I mentioned about `!all` not making much sense in the module file too.

Reply:
Sorted that indent